### PR TITLE
Minor grammatical corrections/suggestions

### PR DESCRIPTION
**Pull request by 0xandreharrison**

### CODE REVIEW

File: Solana_And_Web3/en/Section_4/Lesson_1_Finishing_Touches_Web_App_And_Program.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ It would be super cool if you loved a certain GIF submitted by another user SO M
 
 **Solana's super low gas fee means sending small amounts of money like this actually makes sense.** If you do this, you could even make a version of Patreon or BuyMeACoffee on Solana. Not that crazy. You have all the basic skills now.
 
-Who need Stripe and PayPal when you have a super low-gas fee blockchain that lets you instant payments?!?
+Who need Stripe and PayPal when you have a super low-gas fee blockchain that lets you make instant payments?!?
```
Comment (adilanchian) [inline Solana_And_Web3/en/Section_4/Lesson_1_Finishing_Touches_Web_App_And_Program.md:48]: Small typo here as well - 

"Who needs Stripe and PayPal when you have a super low, gas fee blockchain that lets you make instant payments?!?

**Review by adilanchian — CHANGES_REQUESTED** [review]
One minor tweak please :)

**Comment by adilanchian** [comment]
Sup @0xandreharrison !

Just checking in :)

File: Solana_And_Web3/en/Section_4/Lesson_1_Finishing_Touches_Web_App_And_Program.md (hunk 1 above)
Comment (adilanchian) [reply Solana_And_Web3/en/Section_4/Lesson_1_Finishing_Touches_Web_App_And_Program.md]: I went ahead and resolved this :)

**Comment by 0xandreharrison** [comment]
Sup @adilanchian! I'll see you at the next Solana NFT collection project :)

**Comment by 0xandreharrison** [comment]
I hope you weren't waiting for me! If so, I'm sorry not too familiar with how all the processes work ;)

**Pull request merged**